.github/workflows/pyside6.yml

```diff
@@ -0,0 +1,34 @@
+name: Run PySide6 Tests
+
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            VFXPLATFORM: "2018"
+            PYTHON: "3.6"
```

Comment:
## I think, the `strategy:` block can go away.

A `strategy: matrix:` block is used to execute tests within the Docker container multiples times in a row [in `run-tests.yml`](https://github.com/mottosso/Qt.py/blob/master/.github/workflows/run-tests.yml#L14), using different parameters.

→ I think this does not have any use in here, and can be removed.